### ISSUE TITLE
Odd Incompatibility with Lios Overhauled Villages

### ISSUE THREAD

**Issue by Keoxxa** [hammertater/treechop#298]
For some reason, with **Lios Overhauled Villages** installed, tree chopping breaks. I can chop any logs attached to a leaf or not (including ones I place myself) even with settings enabled to check for leaves, and no matter how many chops I do the tree won't fall. (11 chops of 6, for example)

Not sure what causes it at all, but it seems to only be that mod causing the issue. Once I disabled that mod, treechop worked again.

**Comment by hammertater** [comment]
Weird! Could you share your latest.log file?

**Comment by Keoxxa** [comment]
I just spent the last 4-5 hours removing and messing with the modpack to figure things out, and somehow I got things to work.
The logs didn't provide anything helpful, I went through them myself.
I'm guessing it had something to do with v-tweaks, as disabling the felling part of that mod is what fixed things... I think. 
(I've been trying to figure out what was felling trees for days now)
It was weird, though. Once I added Lios mod, only then everything broke. Guess it was the straw that broke the camels back or something. No idea.

Anyway, the issue seems fixed now. I'll reopen this or add more if it happens again.
Apologies for the concern.
